shared/chat/conversation/messages/coinflip/index.tsx

```diff
@@ -83,9 +82,6 @@ class CoinFlip extends React.Component<Props, State> {
             <Kb.Text selectable={true} type="BodySmallSemibold">
               {!Styles.isMobile && 'Collecting '}secrets: {this._revealSummary()}
             </Kb.Text>
-            {this.props.phase === 'complete' && (
```

Comment:
@adamjspooner ts catches that this can never be true. this was added about 3 months ago